design-editor/src/pane/design-editor-element.js

```diff
@@ -649,14 +649,69 @@ class DesignEditor extends DressElement {
 		}
 	}
 
+	/**
+	 * Find TAU Components related to indicated element
+	 * @param {HTMLElement} element
+	 * @protected
+	 */
+	getComponentsByElement(element) {
+		const packages = packageManager.getPackages(Package.TYPE.COMPONENT)._packages;
```

Comment:
_packages seems to be private member of packages collection.
In content-manager/src/main.js:21  there is a get(packageName) method implemented.

Reply:
Sorry, I thought this PR is in review :)

Reply:
Thank you for constructive suggestion.